addon/utils/wrap-children.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Utility to wrap nodes in wrapper nodes
+ *
+ * @param {Array} nodesToWrap   List of nodes to be wrapped
+ * @param {HtmlNode} wrapper    The node to wrap matching child nodes in.
+ *
+ */
+export default function wrapChildren(nodesToWrap, wrapper) {
+  let clonedWrapper;
+
+  nodesToWrap.forEach((node) => {
+    clonedWrapper = wrapper.cloneNode();
+    node.parentNode.insertBefore(clonedWrapper, node);
+    clonedWrapper.appendChild(node);
+  });
+}
+
+/**
+ * Convert htmlCollection to an array in order
+ * to use array methods on the collection.
+ */
+export function collectionToArray(htmlCollection) {
```

Comment:
FYI @sivakumar-kailasam splatting an `htmlCollection` into an array wasn't working as expected, so that's what this is here for.

Reply:
interesting, what did you notice?

Reply:
Turns out I was mistaken. The original reason this wasn't working wasn't because of splatting this into an array, it was because I was calling `reject` on a native array. So yeah I'll update.